### DISCUSSION TITLE
Webhook Secret Validation Failure

### DISCUSSION THREAD

**Discussion by TomerFi** [community/community#107789] [Apps, API and Webhooks] [answered]
### Select Topic Area

Bug

### Body

Deploy Keys and Private Access Tokens have disappeared in multiple projects for the last few days. I replaced all the missing, but ever since yesterday, I wonder if it's related. A GitHub App I own stopped working, and in the logs, I see failures to validate the Webhook Secret. Replacing the Webhook Secret - didn't work.

**Comment by AaronDewes** [comment, marked as answer]
The validation issue is a Probot/Octokit issue and not related to GitHub: https://github.com/probot/probot/issues/1974.

I'm not sure about the deploy keys/access tokens thing.

**Reply by TomerFi** [reply]
The deploy keys and pats might be related to some expiration date or something, although I didn't get any notifications for the expirations. Regardless, this is quickly resolved by creating new keys. For the webhook secret issue, I tried the solution suggested in  https://github.com/probot/probot/issues/1974. Unfortunately, that didn't fix my issue.
